sdk/communication/AzureCommunication/Source/CommunicationCloudEnvironment.swift

```diff
@@ -28,24 +28,21 @@
 #endif
 import Foundation
 
-@objcMembers public class  CommunicationCloudEnvironment {
+@objcMembers public class  CommunicationCloudEnvironment: NSObject {
     private(set) var environmentValue: String
 
     public static let Public = CommunicationCloudEnvironment(environmentValue: "public")
     public static let Dod = CommunicationCloudEnvironment(environmentValue: "dod")
     public static let Gcch = CommunicationCloudEnvironment(environmentValue: "gcch")
 
-    init(environmentValue: String) {
+    public init(environmentValue: String) {
         self.environmentValue = environmentValue
     }
 
     static func fromModel(environmentModel: CommunicationCloudEnvironmentModel) -> CommunicationCloudEnvironment {
         return CommunicationCloudEnvironment(environmentValue: environmentModel.requestString);
     }
-}
 
-// CommunicationCloudEnvironment Extension
-extension CommunicationCloudEnvironment: Equatable {
```

Comment:
if you remove `CommunicationCloudEnvironment: Equatable `, will the == func gets called ?
when we do 'aCommunicationCloudEnvironment == anotherCommunicationCloudEnvironment`?

Reply:
The implementation of `==` in swift will not be effected. But this implementation will not exist for ObjC

Reply:
XCode gives us a warning saying we should implement `isEqual`

![image](https://user-images.githubusercontent.com/8701731/106640852-67918d00-653b-11eb-99e2-c77f65409087.png)

Reply:
And should we move the static func out for PhoneNumberIdentifier as well?

Reply:
My mistake. We should leave the == as is.